nemo_rl/utils/logger.py

```diff
@@ -69,9 +73,11 @@ class GPUMonitoringConfig(TypedDict):
 class LoggerConfig(TypedDict):
     log_dir: str
     wandb_enabled: bool
+    swanlab_enabled: bool
     tensorboard_enabled: bool
     mlflow_enabled: bool
     wandb: WandbConfig
+    swanlab: SwanlabConfig
```

Comment:
I think this should be NotRequired since you don't always include this dict
```suggestion
    swanlab: NotRequired[SwanlabConfig]
```